packages/web-react/src/types/shared/dictionaries.ts

```diff
@@ -28,6 +33,10 @@ export type EmotionColorsDictionaryType<C = undefined> = (typeof EmotionColors)[
 export type TextColorsDictionaryKeys = keyof typeof TextColors;
 export type TextColorsDictionaryType<C = undefined> = (typeof TextColors)[TextColorsDictionaryKeys] | C;
 
+/* Direction */
+export type DirectionDictionaryKeys = keyof typeof Direction;
```

Comment:
This also isn't in our shared dictionaries. We might add that, but it should be discussed with Michal too, because he has to be aware of that and name the props in Figma like that.

Reply:
fixed